magma/interface.py

```diff
@@ -1,9 +1,8 @@
 from itertools import chain
 from collections import OrderedDict
 from .conversions import array
-from .ref import AnonRef, InstRef, DefnRef, LazyDefnRef
-from .t import Type, Kind, MagmaProtocolMeta
-from .port import INPUT, OUTPUT, INOUT
+from .ref import AnonRef, InstRef, DefnRef, LazyDefnRef, ArrayRef, TupleRef, NamedRef
```

Comment:
looks like `AnonRef`, `ArrayRef`, and `TupleRef` imports are not used in this file